vraptor-core/src/main/java/br/com/caelum/vraptor/serialization/gson/DateGsonConverter.java

```diff
@@ -41,25 +41,29 @@
 @Dependent
 public class DateGsonConverter implements JsonDeserializer<Date>, JsonSerializer<Date>{
 
-	private final SimpleDateFormat iso8601Format;
+	private final SimpleDateFormat format;
 
 	public DateGsonConverter() {
-		this.iso8601Format = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssZ");
+		this.format = getDateFormat();
 	}
 	
 	@Override
 	public JsonElement serialize(Date date, Type typeOfSrc, JsonSerializationContext context) {
-		String dateString = iso8601Format.format(date);
+		String dateString = format.format(date);
 		return new JsonPrimitive(dateString);
```

Comment:
[...] and here use `getFormat().format(date)`. We need to create a new instance of `SimpleDateFormat` because its not thread-safe.